dotnet/src/Connectors/Connectors.Memory.Weaviate/WeaviateMemoryStore.cs

```diff
@@ -563,11 +563,11 @@ private static MemoryRecordMetadata ToMetadata(WeaviateObject weaviateObject)
 
         return new(
             false,
-            string.Empty,
-            weaviateObject.Properties["sk_id"].ToString(),
-            weaviateObject.Properties["sk_description"].ToString(),
-            weaviateObject.Properties["sk_text"].ToString(),
-            weaviateObject.Properties["sk_additional_metadata"].ToString()
+            "",
```

Comment:
nit. IMHO I like to keep `string.Empty`, as it is easy to spot and gives clarity of the intention. For me it also make me think twice as more when adding those.